src/MOCK_DB_DATA/library-items/treeview/angular-tree-component.ts

```diff
@@ -2,9 +2,9 @@ import { LibraryItemIds } from '../library-item-ids';
 import { LibraryItem, LibraryItemType } from 'src/app/views/shared/models/library-item.model';
 import { LibraryItemConstants as CONST } from '../library-item-constants';
 
-export const TreeViewBuilder: LibraryItem[] = [{
-  id: LibraryItemIds.TreeViewBuilder,
-  title: 'Tree view builder',
+export const AngularTreeComponent: LibraryItem[] = [{
+  id: LibraryItemIds.AngularTreeComponent,
+  title: 'angular tree component',
```

Comment:
The thing is that Circlon has named it in all lowercase, and since this is an imprementation of that component it would make sense to name it in the same manner?

Reply:
Hmm.... I don't agree. When used as a title I think we should use our conventions. And Circlon use uppercase first letter when contex is a title, se heading here: https://circlongroup.github.io/angular-tree-component/